test/unit/lucid.relations.spec.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
I've just changed them, IMO this is a problem with chai and standard compatibility. Many of the chai assertion methods are now classes as "unused expressions" and it is a shame that they can't be used now.